src/index.ts

```diff
@@ -45,7 +45,7 @@ const renderWithCarbon = ({
   changeSortOrder,
 }: RenderWithCarbonArguments) => {
   if (env.carbon && changeSortOrder && theme) {
-    render(rootElement, { layout, model, manageData, theme, selectionsAPI, changeSortOrder, app, rect });
+    render(rootElement, { layout, model, theme, selectionsAPI, changeSortOrder, app, rect });
```

Comment:
I'm pretty sure they use the instance of manageData that is passed here. If you remove it here you would have to change so the native implementation imports the function from file instead

Reply:
ops, that's not my intention to remove managedata. Thanks for finding it out.